### ISSUE TITLE
autoexec env feature

### ISSUE THREAD

**Issue by mxrlkn** [kaimallea/csgo#35]
Just like I can add SM_PUGSETUP_* to env and set pug settings, It would be nice to be able to do the same with autoexec.cfg.

Maybe something like
`--env "AUTOEXEC_SV_COACHING_ENABLED=1"`

Currently I'm a bit unsure where to set those manually as autoexec gets overriden.

**Comment by Soren90** [comment]
Maybe this will solve your issue? #45

**Comment by mxrlkn** [comment]
Yeah. I do like how you can set the pugsetup settings via en env variable. But at least with your changes, you can manually change autoexec now.

@kaimallea Feel free to close this issue if you don't think it's necessary.
